src/core/MediaPlayer.h

```diff
@@ -68,19 +69,25 @@ Q_OBJECT
         \brief Returns libvlc media player object.
         \return libvlc media player (libvlc_media_player_t *)
     */
-    libvlc_media_player_t *core();
+    libvlc_media_player_t *core() const;
```

Comment:
const keyword can be applied since we're returning only a private member of the class